pt/news/_posts/2016-09-08-ruby-2-4-0-preview2-released.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: news_post
+title: "Lançado o Ruby 2.4.0-preview2"
+author: "naruse"
+translator: "Filipe Rocha"
+date: 2016-09-08 18:00:00 +0900
+lang: pt
+---
+
+Temos o gosto de anunciar a versão 2.4.0-preview2 do Ruby.
+
+A versão 2.4.0-preview2 do Ruby é a segunda *preview* do Ruby 2.4.0,
+lançada com o fim de obter *feedback* da comunidade.
+Poderão
```

Comment:
Instead of `Poderão`, I  think `Sintam-se confortáveis` is closer to the original one.